### DISCUSSION TITLE
v1.5.0

### DISCUSSION THREAD

**Discussion by marcus-crane** [marcus-crane/october#91] [Announcements]
This release brings support for Linux systems which means that October is available for all major platforms.

![linux_selector_light](https://github.com/marcus-crane/october/assets/14816406/16e813f0-4aeb-42d3-9cc3-a06150a51f3b)

To celebrate this milestone, I've given the docs site a bit of a content refresh although there are still more changes I intend to make.

There were also a few other small changes:

- Fixed pesky sizing issue that means you could scroll down slightly on most screens. The Windows settings screen specifically may still have this issue depending on your screen resolution
- Updated artifact file format naming to follow a consistent standard
- Slightly increased the default window width for October to give it a little more breathing room

**Full Changelog**: https://github.com/marcus-crane/october/compare/v1.4.0...v1.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/marcus-crane/october/releases/tag/v1.5.0'>v1.5.0</a>.</em>

**Comment by statmonkey** [comment]
Of all the different tools I have tried to use this is the one. A total gamechanger. Thank you.

**Reply by marcus-crane** [reply]
Oh! Thanks for your kind words 😀 

I haven't done much work on it lately as I've got some life stuff going on (moving house mainly) but I do intend to get back to it shortly.

On my side, I just see all the sharp edges with it so hopefully we get to a much nicer polished 2.0 someday 😅

**Comment by statmonkey** [comment]
It does what it is supposed to very reliably. I have several Kobo's and
getting my stuff into Obsidian was a convoluted pain before I found
October. The edges look not so rough if I am not distracted in my reading
wondering if the notes I am taking are going to be available later and this
gives me that assurance. It's polished enough for me :) Hope the move goes
well, maybe you can have a beer on me!
